distribution/bin/enso

```diff
@@ -1,5 +1,5 @@
 COMP_PATH=$(dirname "$0")/../component
 
 JAVA_OPTS="--add-opens=java.base/java.nio=ALL-UNNAMED $JAVA_OPTS"
-exec java --module-path $COMP_PATH $JAVA_OPTS -m org.enso.runtime/org.enso.EngineRunnerBootLoader "$@"
+exec java --module-path $COMP_PATH $JAVA_OPTS -m org.enso.runner/org.enso.runner.Main "$@"
```

Comment:
Now the runner is on `--module-path` without a single line of change in `org.enso.runner.Main` - interesting. We'll see what happens with this when #10010 gets closer.